i18n/fr-FR.yml

```diff
@@ -4,10 +4,86 @@ _name: Unofficial French Translations!
 components:
   DefaultItinerary:
     clickDetails: Cliquez pour afficher les détails
-    departureArrivalTimes: "{startTime}—{endTime}"
     multiModeSummary: "{accessMode} + {transitMode}"
-    tripDurationFormatZeroHours: "{minutes, number} mn"
-    tripDurationFormat: "{hours, number} h, {minutes, number} mn"
+  ItinerarySummary:
+    fareCost: "{useMaxFare, select, 
+      true {{minTotalFare} - {maxTotalFare}}
+      other {{minTotalFare}}
+    }"
+  NarrativeItinerariesHeader:
+    numIssues: "{issueNum, number} problèmes"
+    resultText: "{pending, select,
+      true {Recherche de vos options en cours...}
+      other {
+        {itineraryNum, number} {itineraryNum, plural,
+          one {trajet trouvé}
+          other {trajets trouvés}
+        }
+      }
+    }"
+    selectArrivalTime: Heure d'arrivée
+    selectBest: Meilleure option
+    selectCost: Prix
+    selectDepartureTime: Heure de départ
+    selectDuration: Durée
+    selectWalkTime: Temps de marche
+    titleText: "{pending, select, 
+      true {Recherche de vos options en cours...}
+      other {
+        {itineraryNum, number} {itineraryNum, plural,
+          one {trajet}
+          other {trajets}}
+        {issueNum, plural,
+          =0 {trouvé}
+          one {(et {issueNum, number} problème) trouvé}
+          other {(and {issueNum,number} problèmes) trouvés}
+        }
+      }
+    }"
+    viewAll: Voir toutes les options
+  PlanFirstLastButtons:
+    # Note to translator: these values are width-constrained.
+    first: Premier
+    last: Dernier
+    next: Suivant
+    previous: Précédent
+  RealtimeAnnotation:
+    ignoreServiceDelays: Appliquer les retards
+    delaysNotShownInResults: "Vos trajets recherchés sont perturbés par des retards.
+      Ces retards ne sont pas pris en compte dans les temps de trajet ci-dessous."
+    delaysShownInResults: "Vos trajets recherchés ont été mis à jour avec les conditions en temps réel.
+      En temps normal, ce trajet prendrait <b>{normalDuration}</b>
+      en empruntant les lignes: {routes}."
+    ignoreServiceDelays: Ignorer les retards
+    serviceUpdate: Information sur le service
+  SaveTripButton:
+    cantSaveText: Impossible d'enregistrer
+    cantSaveTooltip: Seuls les trajets en transports en commun sans location de véhicules et sans course en voiture peuvent être suivis.
+    saveTripText: Enregistrer
+    signInText: Connectez-vous pour enregistrer
+    signInTooltip: Veuillez vous connecter pour enregistrer ce trajet.
+  SimpleRealtimeAnnotation:
+    usingRealtimeInfo: Ce trajet utilise les informations en temps réel sur le trafic et les retards
+  TabbedItineraries:
+    optionNumber: "Option {optionNum,number}"
```

Comment:
```suggestion
    optionNumber: "Option {optionNum, number}"
```

Reply:
addressed in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/ad8ea3326848a5ac077e01898d7f7657412fda84